rgw/v2/tests/s3cmd/configs/test_large_omap_clearence_brown_field_pre_upgrade.yaml

```diff
@@ -0,0 +1,11 @@
+# Polarian: CEPH-83591446
+# Script: test_large_omap.py
+config:
+  haproxy: true
+  user_count: 1
+  bucket_count: 1
+  objects_count: 20
+  test_ops:
+    large_omap: true
+    cluster_is: brownfield
```

Comment:
Should this be brownfield or greenfield?

Also please correct the typo of the config name (clearance not clearance)

Reply:
need this as brownfield only as it used in upgrade suite
will update the typo